docs/docs/webchat/webchat-connectivity.md

```diff
@@ -0,0 +1,53 @@
+---
+id: webchat-connectivity
+title: Connectivity
+---
+
+Sometimes, the connection can be lost between the server and the chatbot. To handle this situation, you can set specific parameters to quickly detect the connection issue.
+
+For a better user experience, you can also warn the user by configuring a customized error message to be displayed within the webchat interface.
+
+To do so:
+
+1. Set the `Botonic.isOnline()` method. Values are `true` or `false` depending on your server configuration.
```

Comment:
Change `To do so` above with the mentioned suggestion or similar and remove these two points. The first one is not accurated.